src/web/badge/src/lib.rs

```diff
@@ -166,17 +168,17 @@ mod tests {
     }
 
     #[test]
-    #[ignore]
     fn test_to_svg() {
-        use std::fs::File;
-        use std::io::Write;
+        use std::{fs::File, io::Write};
+
         let mut file = File::create("test.svg").unwrap();
         let options = BadgeOptions {
             subject: "build".to_owned(),
             status: "passing".to_owned(),
             ..BadgeOptions::default()
         };
         let badge = Badge::new(options).unwrap();
+
         file.write_all(badge.to_svg().as_bytes()).unwrap();
```

Comment:
I think a better idea might be to write the badge to memory instead of to disk. It would also be nice to compare it to a known good badge, but it would be fine even without.